### DISCUSSION TITLE
Stage Manager and Rectangle Pro

### DISCUSSION THREAD

**Discussion by jdh313** [rxhanson/RectanglePro-Community#306] [Q&A] [answered]
When using stage manager I've noticed that if I maximize a window with a snap action it will leave the space on the side for the recent apps, but if I maximize via a window throw it does not. Is there a setting I'm missing or something I'm doing wrong? I've attached my configuration file below. 

Thanks!
[RectangleProConfig.txt](https://github.com/rxhanson/RectanglePro-Community/files/11982762/RectangleProConfig.txt)

**Comment by rxhanson** [comment, marked as answer]
Thanks for pointing this out. This was a discrepancy that I had left in when adding in the Stage Manager functionality, as I wasn't really sure what I wanted to do at that time. I'll adjust this in the next release.

**Reply by jdh313** [reply]
Awesome, thank you!

**Reply by rxhanson** [reply]
Ok, got a chance to dig through this and I think it's actually a bug. Do you have a specific scenario where it doesn't work for you, maybe on a second monitor or something unusual about your setup?

(Upon looking back through the code, the window throw is using the same logic as the other actions for stage manager detection at this time, and I can't reproduce it after having loaded your config)
